### PR TITLE
collaborative code editor guided project added

### PR DESCRIPTION
**Pull request by oneknucklehead**
Added the steps guiding the user through the project.

### CODE REVIEW

File: collaborative-code-editor/spec.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+name: Collaborative code editor
+tags: ["ReactJS",'Socket.io']
+starter-files: https://github.com/oneknucklehead/collaborative-editor
+type: frontend
```
Comment (Pranavtechie) [inline collaborative-code-editor/spec.yml:4]: You can convert this to fullstack as your also building the server.

**Review by Pranavtechie — CHANGES_REQUESTED** [review]
I've also made 2 PRs in the Starter file repo. Please merge them. 

Also you can move the package.json file into the server folder, so that the client and server files are isolated completely. (Once done, please update the .cdmrc accordingly as well) 

You can also reduce the boilerplate code in the server.js file and instruct those changes in the project instructions. 

You can also elaborate a little hard parts of the instructions by guiding users with small code snippets.

**Review by oneknucklehead — COMMENTED** [review]
Made the said changes

**Comment by oneknucklehead** [comment]
Hey! @Pranavtechie just following up on the PR, i had made the said changes. If things are all okay, we can gtm

**Comment by Pranavtechie** [comment]
@oneknucklehead Please share your email address here or reach out to me at [pranav@codedamn.com](mailto:pranav@codedamn.com)

**Comment by oneknucklehead** [comment]
> @oneknucklehead Please share your email address here or reach out to me at [pranav@codedamn.com](mailto:pranav@codedamn.com)

Here's my email: zohebcool1542@gmail.com

**Pull request merged**